docs/runbooks/VirtControllerRESTErrorsBurst.md

```diff
@@ -2,19 +2,46 @@
 
```

Comment:
GLOBAL: Please add your name and date in comment to all modules that you update. Thanks!